### ISSUE TITLE
Problems with duping item and item loss.

### ISSUE THREAD

**Issue by ArifErdoganK** [dig/ender-vaults#22]
We are utilizing the multi-server feature, with the setting that loads storage data within 10 seconds upon each server change. After depositing an item into the storage within any of the multi-servers, we immediately switch to a different server, retrieve the item from the storage, quickly return to the server where we initially deposited the item, and are able to retrieve it again from the storage. This process results in item duplication, or "duping". When I personally tested this, duplication did not occur, but the storage was cleared. However, some players experience item duplication or storage clearance issues.

Video: https://youtu.be/em8SIhazpso
config.yml: https://hastebin.com/share/siqujorure.yaml

While our players continue to experience item loss, a dupe issue has also emerged. We have been mentioning this in the comments on SpigotMC for a long time, but it seems that it is not being taken into consideration.

I hope you can resolve this issue quickly. Thank you.

**Comment by ArifErdoganK** [comment]
I also experienced duping. The reason behind this is that when I logged into the server, instead of waiting for the 10-second delay for the storage to open, I didn't wait. The storage opens directly, and I can take any item from it. Then, when I switch servers and open the storage again, I have to wait for the 10-second delay. After waiting for 10 seconds and opening the storage, the item reappears. The storage opening delay is set to 10 seconds on all servers. Sometimes it malfunctions and opens directly.

**Comment by msmanui** [comment]
Hello. What happend in the end? You still using this plugin or do you have any better alternative that support mysql?

**Comment by ArifErdoganK** [comment]
> Hello. What happend in the end? You still using this plugin or do you have any better alternative that support mysql?

Hello, I’ve been using CocoVaults for a while and I’m satisfied.

**Comment by msmanui** [comment]
> > Hello. What happend in the end? You still using this plugin or do you have any better alternative that support mysql?
> 
> Hello, I’ve been using CocoVaults for a while and I’m satisfied.

Yeah. I found it some days ago and i asked to add EnderVault migration support, so i will be swithching soon. Thanks for your answer.
